parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -249,6 +249,12 @@ fi
 if [[ "${MESSAGE_OF_THE_DAY}" != "" ]]; then
     if isMarinerOrAzureLinux "$OS" && [ -f /etc/dnf/automatic.conf ]; then
       sed -i "s/emit_via = motd/emit_via = stdio/g" /etc/dnf/automatic.conf
+    elif [[ $OS == "$UBUNTU_OS_NAME" ]] && [[ -d "/etc/update-motd.d" ]]; then
+          aksCustomMotdUpdatePath=/etc/update-motd.d/99-aks-custom-motd
+          touch "${aksCustomMotdUpdatePath}"
+          chmod 0755 "${aksCustomMotdUpdatePath}"
+          echo '#!/bin/bash' >> "${aksCustomMotdUpdatePath}"
```

Comment:
can we just use `cat` or `tee` to write the script out in a single call? feel like multiple echo statements might also be more error prone

Reply:
also, can we name it accordingly if it's actually a bash script? e.g. aksCustomMotdUpdateScript

I might be missing something though if it's being named this way, also should we be using `.sh` as the file extension? looks like right now we don't use any extension

Reply:
Naming needs to follow the convention described here: https://manpages.ubuntu.com/manpages/oracular/en/man5/update-motd.5.html

Will pass the whole thing in one call.

Reply:
oh lol I just looked at the PR description, I guess in the docs they don't use `.sh`...

Reply:
Other names that might be added by Ubuntu can for example be `10-help-text` or `50-landscape-sysinfo`